package.json

```diff
@@ -21,7 +23,10 @@
   "dependencies": {
     "@atlaskit/pragmatic-drag-and-drop": "^1.5.2",
     "@atlaskit/pragmatic-drag-and-drop-hitbox": "^1.0.3",
-    "@iconify/vue": "^4.3.0"
+    "@iconify/vue": "^4.3.0",
+    "oxc-parser": "^0.62.0",
+    "pathe": "^2.0.3",
+    "zod": "^3.24.2"
```

Comment:
Shouldn't these be devDependencies?

Reply:
Yes they should. I have made the necessary changes.